src/github.com/getlantern/flashlight/logging.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/dogenzaka/rotator"
+	"github.com/getlantern/appdir"
+	"github.com/getlantern/golog"
+	"github.com/getlantern/wfilter"
+	"github.com/segmentio/go-loggly"
+)
+
+var (
+	LogTimestampFormat = "Jan 02 15:04:05.000"
+	logglyToken        = "469973d5-6eaf-445a-be71-cf27141316a1"
```

Comment:
I think we should make this something that's passed in at compile time, much like version and buildDate (see crosscompile.bash).  The reason is that if someone forks flashlight, we don't want them reporting to the same Loggly account that we use.